_search-plugins/search-pipelines/explanation-processor.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: default
+title: Hybrid score explanation
+nav_order: 15
+has_children: false
+parent: Search processors
+grand_parent: Search pipelines
+---
+
+# Hybrid score explanation processor
+Introduced 2.19
+{: .label .label-purple }
+
+The `hybrid_score_explanation` response processor adds the normalization and combination results to the returned search response. For more information, see [`hybrid` query]({{site.url}}{{site.baseurl}}/query-dsl/compound/hybrid/) that is the debugging tool that provides insights of score normalization processing.  
```

Comment:
I'm not following the second sentence here. Please rephrase or delete the text following the link.